src/tls.c

```diff
@@ -176,17 +175,51 @@ enftun_tls_handshake(struct enftun_tls* tls)
 }
 
 int
-enftun_tls_connect(struct enftun_tls* tls, const char** hosts, const char* port)
+enftun_tls_connect(struct enftun_tls* tls, const char** hosts, const char* port, int fwmark)
 {
     int rc;
 
-    rc = tls->sock->ops.connect_any(tls->sock, hosts, port);
-    if (rc < 0)
-        goto out;
+    enftun_log_info("%s Enter tls->sock=%p tls->sock_type=%d\n",
```

Comment:
This should be a debug, not info comment.  (Or removed entirely.)

Reply:
I didn't intend on this comment being merged.